wdl2cwl/WdlV1_1.py

```diff
@@ -0,0 +1,203 @@
+import sys
```

Comment:
Whoops, this files snuck back in. I renamed `wdl2cwl/WdlV1_1.py` into `wdl2cwl/main.py` ; so make sure that you have everything you need in `main.py` and remove `wdl2cwl/WdlV1_1.py` from this PR